### PR TITLE
Fix Homepage, website font and post page layout

### PR DESCRIPTION
**Pull request by 526avijitgupta**
* Homepage (centered avatar)
* Replace `Roboto` with `Quicksand-Medium` for font
* Add letter spacing for content to look more readable
* Post page layout - better spacing

### CODE REVIEW

File: themes/gokarna/layouts/index.html [hunk 1]
```diff
@@ -1,40 +1,9 @@
...
-
-At ver
-  </p>
-</div>
+<section class="home-about">
+	<div class="avatar">
+		<img alt="avatar" src="https://www.startpage.com/av/proxy-image?piurl=https%3A%2F%2Fencrypted-tbn0.gstatic.com%2Fimages%3Fq%3Dtbn%3AANd9GcSglXjS92JzGJeFHp4iwpFnP9RbjXnBe7DR2hDg3PSe_n3UBw%26s&sp=1625351290T763370d97b7d0cb466789c2d5f5e23a6dd4674cb572fc6450eddfef3d4b60370">
```
Comment (yashmehrotra) [inline themes/gokarna/layouts/index.html:4]: No image shows up, better to add a normal link in config.toml and refer from that

**Review by yashmehrotra — CHANGES_REQUESTED** [review]

File: themes/gokarna/layouts/index.html [hunk 2]
```diff
@@ -1,40 +1,9 @@
...
+	</div>
+	<h1>Chaman Chutiya</h1>
```
[the remark this answers is not in the document]
Comment (526avijitgupta) [reply themes/gokarna/layouts/index.html:6]: Error aa rha hai. Templating vala tu hi dekh bro :)

File: themes/gokarna/static/css/main.css [hunk 3]
```diff
@@ -1,7 +1,9 @@
 html {
   background: white;
   color: black;
-  font-family: "Roboto", sans-serif;
+  font-family: "Quicksand", sans-serif;
+  font-weight: 300;
```
[the remark this answers is not in the document]
Comment (526avijitgupta) [reply themes/gokarna/static/css/main.css:5]: I had deliberately used 300. Looked much better

[the remark this answers is not in the document]
Comment (526avijitgupta) [reply themes/gokarna/static/css/main.css:5]: 400 ke cramped up aur bold lag rha hai

[the remark this answers is not in the document]
Comment (526avijitgupta) [reply themes/gokarna/static/css/main.css:5]: what do you think?

File: themes/gokarna/layouts/index.html (hunk 1 above)
Comment (526avijitgupta) [reply themes/gokarna/layouts/index.html:4]: Yeh bhi kar de. Templating se related hai :p

File: themes/gokarna/static/css/main.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (526avijitgupta) [reply themes/gokarna/static/css/main.css:5]: Especially check the 2nd post page. You'll notice the difference

**Pull request merged**